docs/source/markdown/podman-run.1.md

```diff
@@ -1305,6 +1306,19 @@ The default working directory for running binaries within a container is the roo
 The image developer can set a different default with the WORKDIR instruction. The operator
 can override the working directory by using the **-w** option.
 
+#### **\-\-pidfile**=*path*
+
+When the pidfile location is specified, the container process' PID will be written to the pidfile. (This option is not available with the remote Podman client)
+After the container is started, the location for the pidfile can be discovered with the following `podman inspect` command:
+
+    $ podman inspect --format '{{ .Config.CreateCommand }}' $CID
```

Comment:
Not sure what you are trying to do with these two lines.

Reply:
I have replace these lines with pidfile inspect command